examples/react-wizards-i18n/src/spells/languagePicker.ts

```diff
@@ -0,0 +1,67 @@
+import { CONTENT_NODE_SUBMIT, createSpell, INTERVIEW_INTRO_STATE } from "@xstate-wizards/spells";
+
+export const ID_LANGUAGE_PICKER = "languagePicker";
+
+const CONTENT_NODE_SUBMIT_DEFERRED = (t) => ({
+  type: "button",
+  buttonType: "submit",
+  text: t("continue"),
+  event: "SUBMIT",
+  attrs: { size: "lg", width: "100%" },
+  disabledByFreshDelay: true,
+});
+
+export const machineMapping = createSpell({
+  key: ID_LANGUAGE_PICKER,
+  version: "1",
+  config: {
+    initial: INTERVIEW_INTRO_STATE,
+    // title: "abcd",
+    title: (t) => t("title"),
+    exitTo: "/",
+    sectionsBar: [{ name: (t) => t("sectionBar"), trigger: "awpType" }],
+  },
+  editor: {}, // TODO: rmv
+  models: {
+    User: { loader: {} },
+  },
+  schema: {
+    type: "object",
+    properties: {
+      states: {
+        type: "object",
+        properties: {},
+      },
+    },
+  },
+  states: {
+    [INTERVIEW_INTRO_STATE]: {
+      content: (ctx, t) => {
+        return [
+          {
+            type: "h3",
+            text: t("h3"),
+          },
+          { type: "h3", text: t("h1") },
+          { type: "p", text: t("p") },
+          {
+            type: "input",
+            inputType: "text",
+            label: t("label"),
+            assign: { path: "email" },
+            validations: ["required", "validEmail"],
+          },
+          CONTENT_NODE_SUBMIT,
+          CONTENT_NODE_SUBMIT_DEFERRED(t),
```

Comment:
as a heads up @youmustfight, one thing I realized after our discussion is that if we drill the translate function through the content function, we will need functional subcomponents. LMK if you have any thoughts here but I think the trade-off is still worth it

Reply:
Yea, let's go for it. I'm fine with this pattern